pages/flow/index.js

```diff
@@ -33,3 +59,41 @@ export default function Flow() {
 const StyledList = styled.ul`
   list-style: none;
 `;
+
+const StyledListItem = styled.li`
+  display: flex;
```

Comment:
Only for UX: Did you try adding something like `justify-content: space-between;`?
![Bildschirmfoto 2022-10-19 um 18 34 52](https://user-images.githubusercontent.com/71948045/196751429-509ae3d0-87ce-43ac-9715-5c01e4cc2b45.png)

Reply:
Thanks for the hint! UX ist not great yet. Will work on that later (hopefully). But with space-between it already looks better :)